### PR TITLE
Hide Packages section if no build info exists.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

**Comment by duftler** [comment]
@anotherchrisberry please review.

File: app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.controller.js [hunk 1]
```diff
@@ -299,7 +299,7 @@ module.exports = angular.module('spinnaker.loadBalancer.aws.create.controller',
             $scope.loadBalancer.region, listener.sslCertificateType || this.certificateTypes[0]);
         });
       });
-    }
+    };
```
Comment (anotherchrisberry) [inline app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.controller.js]: thanks

File: app/scripts/modules/openstack/serverGroup/details/serverGroupDetails.html [hunk 2]
```diff
@@ -165,10 +165,8 @@ <h3 select-on-dbl-click>
         <dd ng-if="ctrl.serverGroup.image.properties.ramdisk_id">{{ctrl.serverGroup.image.properties.ramdisk_id}}</dd>
       </dl>
     </collapsible-section>
-    <collapsible-section heading="Package">
-      <div ng-if="!(ctrl.serverGroup.buildInfo && ctrl.serverGroup.buildInfo.jenkins)">None.</div>
-      <dl ng-class="ctrl.InsightFilterStateModel.filtersExpanded ? '' : 'dl-horizontal dl-wide'"
-          ng-if="ctrl.serverGroup.buildInfo && ctrl.serverGroup.buildInfo.jenkins">
+    <collapsible-section heading="Package" ng-if="ctrl.serverGroup.buildInfo && ctrl.serverGroup.buildInfo.jenkins">
```
Comment (duftler) [inline app/scripts/modules/openstack/serverGroup/details/serverGroupDetails.html]: @jcwest fyi

File: app/scripts/modules/cloudfoundry/serverGroup/details/serverGroupDetails.html [hunk 3]
```diff
@@ -179,9 +179,8 @@ <h4 class="text-center" ng-if="serverGroup.isDisabled">[SERVER GROUP IS DISABLED
         </div>
       </dl>
     </collapsible-section>
-    <collapsible-section heading="Package">
-      <dl ng-class="InsightFilterStateModel.filtersExpanded ? '' : 'dl-horizontal dl-wide'"
-          ng-if="serverGroup.buildInfo">
+    <collapsible-section heading="Package" ng-if="serverGroup.buildInfo">
```
Comment (duftler) [inline app/scripts/modules/cloudfoundry/serverGroup/details/serverGroupDetails.html]: @gregturn fyi

**Comment by anotherchrisberry** [comment]
LGTM

**Comment by gregturn** [comment]
Yeah! Thanks.

**Pull request merged**